<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDs8</class>
 <widget class="QWidget" name="IDs8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 8 TID/SID</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxRNGInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>RNG Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="1">
       <widget class="TextBox" name="textBoxSeed1"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelInitialAdvances">
        <property name="text">
         <string>Initial Advances</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMaxAdvances">
        <property name="text">
         <string>Max Advances</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSeed1">
        <property name="text">
         <string>Seed 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSeed0">
        <property name="text">
         <string>Seed 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TextBox" name="textBoxInitialAdvances">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="TextBox" name="textBoxMaxAdvances">
        <property name="text">
         <string>100000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxSeed0"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="TableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Profiles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Brilliant Diamond</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonTID">
        <property name="text">
         <string>TID</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="5">
       <widget class="QTextEdit" name="textEditTSVFilter">
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelTSV">
        <property name="text">
         <string>TSV</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonTIDSID">
        <property name="text">
         <string>TID/SID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="5">
       <widget class="QTextEdit" name="textEditFilter">
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonG8TID">
        <property name="text">
         <string>G8TID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>textBoxInitialAdvances</tabstop>
  <tabstop>textBoxMaxAdvances</tabstop>
  <tabstop>textBoxSeed0</tabstop>
  <tabstop>textBoxSeed1</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>radioButtonTID</tabstop>
  <tabstop>radioButtonSID</tabstop>
  <tabstop>radioButtonTIDSID</tabstop>
  <tabstop>radioButtonG8TID</tabstop>
  <tabstop>textEditFilter</tabstop>
  <tabstop>textEditTSVFilter</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
